### ISSUE TITLE
Automatic upgrade not working

### ISSUE THREAD

**Issue by lapo-luchini** [gliaskos/freebsd-chromium#167]
I wasn't expecting auto-upgrade from a pkg… it erroneously says it has been updated and needs a reload:
![image](https://user-images.githubusercontent.com/420454/68941570-7fe68200-07a6-11ea-8deb-4f9fd89bfad4.png)
pressing the button it restarts, but o course with the same exact version (being installed by `pkg` and owned by `root`, of course).
I think the warning should just be disabled.

**Comment by mwolf-rw** [comment]
Yes, the auto updater *should* be disabled completely, I'll have a look from where that message is coming.

**Comment by lapo-luchini** [comment]
BTW: I didn't notice the screenshot was in Italian at first, it translates as «Update almost completed. Restart Chromium to complete the update.».
